app/views/docs/command-line.phtml

```diff
@@ -118,219 +72,44 @@ brew install --HEAD appwrite</code></pre>
-    </tr>
-    <tr>
-        <td>avatars</td>
-        <td>The avatars command aims to help you complete everyday tasks related to your app image, icons, and avatars</td>
-    </tr>
-    <tr>
-        <td>functions</td>
-        <td>The functions command allows you view, create and manage your Cloud Functions</td>
-    </tr>
-    <tr>
-        <td>databases</td>
-        <td>The databases command allows you to create structured collections of documents, query and filter lists of documents</td>
-    </tr>
-    <tr>
-        <td>health</td>
-        <td>The health command allows you to both validate and monitor your Appwrite server's health</td>
-    </tr>
-    <tr>
-        <td>locale</td>
-        <td>The locale command allows you to customize your app based on your users' location</td>
-    </tr>
-    </tbody>
-</table>
-
-<p>Here are a few sample commands to get you started</p>
-
-<p><b>Create User</b></p>
-
-<p>To create a new user in your project, you can use the `create` command. To successfully create a user, make sure your API key is granted with the scope "users.write".</p>
+<h2><a href="/docs/command-line#next-steps" id="next-steps">Next Steps</a></h2>
+<p>Besides utility commands, the CLI can be used to execute commands like a Server SDK.</p>
+<p>
+<a href="/docs/command-line-commands"><i class="icon-angle-circled-right"></i>Find a full list of commands</a>
+</p>
+<p>You can use the CLI to create and deploy functions and collections. Deploy commands allow you to configure your Appwrite project programmatically and replicate functions and collection schemas across Appwrite projects.</p>
+<p>
+<a href="/docs/command-line-deployment"><i class="icon-angle-circled-right"></i>Learn more about deployment</a>
+</p>
+<p>You can choose to use the CLI in a headless and non-interactive mode without the need for config files or sessions. This is useful for CI or scripting use cases.</p>
 
-<div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite users create --userId "unique()" --email hello@appwrite.io --password very_strong_password</code></pre>
-</div>
-<!-- Attach image of output  -->
+<p>
+<a href="/docs/command-line-ci"><i class="icon-angle-circled-right"></i>Learn more about CI mode</a>
+</p>
 
-<p><b>List Users</b></p>
+<h2><a href="/docs/command-line#help" id="help">Help</a></h2>
 
-<p>To get a list of all your project users, you can use the `list` command. To successfully view the list, make sure your API key is granted with the scope "users.read".</p>
+<p>If you get stuck anywhere, you can always use the <code>help</code> command to get the usage examples.</p>
 
-<div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite users list</code></pre>
-</div>
+<h2><a href="/docs/command-line-ci#configuration" id="configuration">Configuration</a></h2>
```

Comment:
Would it be too much to put Configuration before Next Steps?

Reply:
I'm gonna leave this here. I think they should not worry about config until they explore the CLI. This is a more advanced step for me.